src/Notification.jsx

```diff
@@ -23,7 +23,19 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+let last_error = "";
```

Comment:
This could be array to really just log `if new`.

Reply:
Yes, but I think we really want `log_error_if_changed`, no?  We only want to filter out repeated output from re-rendering the currently open dialog.  I rename the function.